solr/core/src/java/org/apache/solr/util/plugin/NamedListInitializedPlugin.java

```diff
@@ -25,5 +25,10 @@
  * @since solr 1.3
  */
 public interface NamedListInitializedPlugin {
-  void init( @SuppressWarnings({"rawtypes"})NamedList args );
+  /**
+   * <code>init</code> will be called just once, immediately after creation.
+   * <p>The args are user-level initialization parameters that
+   * may be specified, usually in solrconfig.xml
+   */
+  default void init(NamedList<?> args) {}
```

Comment:
I'm confused which side of the argument you actually land on

Reply:
Your change is fine, given how it's used today.  I was contemplating a different future where this interface is totally opt-in, and I'm not sure since it's rather common for plugin abstractions to declare such an init method any way.